docs/stellar-core.1

```diff
@@ -0,0 +1,389 @@
+.\" Automatically generated by Pandoc 1.16.0.2
```

Comment:
I *think* you can actually remove pandoc from the image as it's a non fatal error for xdrpp